core/spec/models/spree/promotion_spec.rb

```diff
@@ -672,17 +672,6 @@
       context 'when the order is complete' do
         it { is_expected.to be true }
 
-        context 'when the promotion was not eligible' do
```

Comment:
@KacperMekarski why are we removing this test?

Reply:
@damianlegawiec because `one_use_per_user` promotion is eligible for all products and we are only checking whether it was used in any of user orders.

Reply:
@damianlegawiec however we should keep this because promotion can be for example expired or some products can be blacklisted.

Reply:
@damianlegawiec I brought it back.